### PR TITLE
To chat massreplaces

### PR DESCRIPTION
**Pull request by SpeedCrash100**

### CODE REVIEW

**Comment by FireFlashie** [comment]
😨😱🤡😭😶

File: code/game/sound.dm [hunk 1]
```diff
@@ -153,12 +153,12 @@ var/const/FALLOFF_SOUNDS = 0.5
...
 			S.environment = A.sound_env
 
-	src << S
+	to_chat(src, S)
 
 /client/proc/playtitlemusic()
 	if(!ticker || !ticker.login_music)	return
 	if(prefs.toggles & SOUND_LOBBY)
-		src << sound(ticker.login_music, repeat = 0, wait = 0, volume = 85, channel = 1) // MAD JAMS
+		to_chat(src, sound(ticker.login_music, repeat = 0, wait = 0, volume = 85, channel = 1) // MAD JAMS)
```
Comment (FireFlashie) [inline code/game/sound.dm]: Grrr

File: code/datums/diseases/pierrot_throat.dm [hunk 2]
```diff
@@ -16,10 +16,10 @@
 	..()
 	switch(stage)
 		if(1)
-			if(prob(10)) affected_mob << "<span class='warning'>You feel a little silly.</span>"
+			to_chat(if(prob(10)) affected_mob, "<span class='warning'>You feel a little silly.</span>")
```
Comment (FireFlashie) [inline code/datums/diseases/pierrot_throat.dm]: GRRRR

File: code/datums/diseases/pierrot_throat.dm [hunk 3]
```diff
@@ -16,10 +16,10 @@
...
 		if(2)
-			if(prob(10)) affected_mob << "<span class='warning'>You start seeing rainbows.</span>"
+			to_chat(if(prob(10)) affected_mob, "<span class='warning'>You start seeing rainbows.</span>")
```
Comment (FireFlashie) [inline code/datums/diseases/pierrot_throat.dm]: GRRRR

File: code/datums/diseases/pierrot_throat.dm [hunk 4]
```diff
@@ -16,10 +16,10 @@
...
 		if(3)
-			if(prob(10)) affected_mob << "<span class='warning'>Your thoughts are interrupted by a loud <b>HONK!</b></span>"
+			to_chat(if(prob(10)) affected_mob, "<span class='warning'>Your thoughts are interrupted by a loud <b>HONK!</b></span>")
```
Comment (FireFlashie) [inline code/datums/diseases/pierrot_throat.dm]: GRRRRRRRRRRRR

File: code/game/gamemodes/gameticker.dm [hunk 5]
```diff
@@ -134,8 +134,8 @@ var/global/datum/controller/gameticker/ticker
 			//Deleting Startpoints but we need the ai point to AI-ize people later
 			if (S.name != "AI")
 				qdel(S)
-		world << "<FONT color='blue'><B>Enjoy the game!</B></FONT>"
-		world << sound('sound/AI/welcome.ogg') // Skie
+		to_world("<FONT color='blue'><B>Enjoy the game!</B></FONT>")
+		sound_to(world, sound('sound/AI/welcome.ogg') // Skie)
```
Comment (FireFlashie) [inline code/game/gamemodes/gameticker.dm]: GRRRRR

File: code/game/machinery/alarm.dm [hunk 6]
```diff
@@ -767,9 +767,9 @@
 	switch(buildstage)
 		if(2)
 			if(istype(W, /obj/item/weapon/screwdriver))  // Opening that Air Alarm up.
-				//user << "You pop the Air Alarm's maintence panel open."
+				to_chat(//user, "You pop the Air Alarm's maintence panel open.")
```
Comment (FireFlashie) [inline code/game/machinery/alarm.dm]: Grrrrrr

File: code/game/machinery/supplybeacon.dm [hunk 7]
```diff
@@ -76,12 +76,12 @@
 	if(expended)
 		return
 	if(surplus() < 500)
-		if(user) user << "<span class='notice'>The connected wire doesn't have enough current.</span>"
+		to_chat(if(user) user, "<span class='notice'>The connected wire doesn't have enough current.</span>")
```
Comment (FireFlashie) [inline code/game/machinery/supplybeacon.dm]: Grrrrr

File: code/game/machinery/supplybeacon.dm [hunk 8]
```diff
@@ -76,12 +76,12 @@
...
 		return
 	set_light(3, 3, "#00CCAA")
 	icon_state = "beacon_active"
 	use_power = 1
-	if(user) user << "<span class='notice'>You activate the beacon. The supply drop will be dispatched soon.</span>"
+	to_chat(if(user) user, "<span class='notice'>You activate the beacon. The supply drop will be dispatched soon.</span>")
```
Comment (FireFlashie) [inline code/game/machinery/supplybeacon.dm]: Grrrrrrr

File: code/game/machinery/supplybeacon.dm [hunk 9]
```diff
@@ -92,7 +92,7 @@
 	set_light(0)
 	use_power = 0
 	target_drop_time = null
-	if(user) user << "<span class='notice'>You deactivate the beacon.</span>"
+	to_chat(if(user) user, "<span class='notice'>You deactivate the beacon.</span>")
```
Comment (FireFlashie) [inline code/game/machinery/supplybeacon.dm]: grrrrrrrrr

File: code/game/machinery/syndicatebeacon.dm [hunk 10]
```diff
@@ -93,7 +93,7 @@
 
 /obj/machinery/power/singularity_beacon/proc/Activate(mob/user = null)
 	if(surplus() < 1500)
-		if(user) user << "<span class='notice'>The connected wire doesn't have enough current.</span>"
+		to_chat(if(user) user, "<span class='notice'>The connected wire doesn't have enough current.</span>")
```
Comment (FireFlashie) [inline code/game/machinery/syndicatebeacon.dm]: grrrrrr

File: code/game/machinery/vending.dm [hunk 11]
```diff
@@ -457,20 +457,20 @@
...
-		usr << "<span class='notice'>You remove the [coin] from the [src]</span>"
+		to_chat(usr, "<span class='notice'>You remove the [coin] from the [src]</span>")
 		coin = null
 		categories &= ~CAT_COIN
 
 	if ((usr.contents.Find(src) || (in_range(src, usr) && istype(src.loc, /turf))))
 		if ((href_list["vend"]) && (src.vend_ready) && (!currently_vending))
 			if((!allowed(usr)) && !emagged && scan_id)	//For SECURE VENDING MACHINES YEAH
-				usr << "<span class='warning'>Access denied.</span>"	//Unless emagged of course
+				to_chat(usr, "<span class='warning'>Access denied.</span>"	//Unless emagged of course)
```
Comment (FireFlashie) [inline code/game/machinery/vending.dm]: grrrr

**Comment by SpeedCrash100** [comment]
Но CI же прошло ;_;

**Pull request merged**